TUTORIAL-ASSETS.md

```diff
@@ -220,15 +218,15 @@ In this above example, we specify a layer of the CAD named `labelLayer` to read
 Assets can also be renamed individually after importing.
 
 
-## Importing
+## Importing your Assets
 
 With your files prepared, you can now begin the extraction process!
 
 - If you're importing GeoJSON directly, you're all set!
 - If you're importing the combined cad.dxf, config.json and georeference.points files, begin by zipping the 3 files together in a .zip file (similar to cad_submission.zip in our examples directory). 
 - Navigate to the [WRLD Indoor Map Designer](https://mapdesigner.wrld3d.com/indoormap/latest/) (login if necessary).
 - Select your Indoor Map from the drop down list and go inside by clicking the Entrance Marker on the map.
```

Comment:
Add in a step "Select the floor you wish to add Assets to using the elevator widget on the right of the screen".  We make reference later to the fact that asset sets are associated with floors, so we should make this obvious up front that the user has to choose a floor before importing.